### PR TITLE
Revert "Add fallback (use backends from resolver if can't reach balan…

### PR DESCRIPTION
**Pull request by kpayson64**
…cer) to grpclb."

This reverts commit aba0a0a54412fe59bc2090334ae26c99bb148097.

This is a test.

### CODE REVIEW

**Comment by markdroth** [comment]
It looks like there's a conflicting file.  I'm guessing that something else got merged since the PR you're trying to roll back here.

File: src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c [hunk 1]
```diff
@@ -1974,7 +1857,6 @@ static grpc_lb_policy *glb_create(grpc_exec_ctx *exec_ctx,
     gpr_free(glb_policy);
     return NULL;
   }
-  grpc_subchannel_index_ref();
```
Comment (markdroth) [inline src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c]: This line should not be removed -- it's unrelated to the PR being reverted here.

Comment (kpayson64) [reply src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c]: Good catch.   Looks like this was an artifact of the upstream merge commit.

**Comment by dgquintas** [comment]
The PR being reverted also caused a memory leak on grpclb_test (see trace [here](https://gist.github.com/dgquintas/42b4d0c7e639d5eea22f7b2ebea030b6)), which can be reproduced via 
```
$ ./tools/run_tests/run_tests.py -l c++ -c asan -r grpclb_test -n inf -S -a 10 --force_use_pollers=epollsig
```
in about 10 seconds.

**Comment by kpayson64** [comment]
#12138
#12510

**Pull request merged**